concrete/src/Permission/Access/Entity/PageOwnerEntity.php

```diff
@@ -29,23 +29,21 @@ public function getAccessEntityUsers(PermissionAccess $pae)
         }
         if (is_object($c) && ($c instanceof Page)) {
             $ui = UserInfo::getByID($c->getCollectionUserID());
-            $users = array($ui);
-
-            return $users;
+            if ($ui) {
+                return [$ui];
```

Comment:
Previously, if the collection author was deleted or there where no page author (?), the function returned `[null]`, now it returns the same as if the page isn't found.